src/components/main/index.js

```diff
@@ -145,8 +159,35 @@ class Main extends React.Component {
             renderNarrativeToggle(this.props.dispatch, this.props.displayNarrative) : null
           }
           {this.props.displayNarrative || this.props.showOnlyPanels ? null : <Info width={calcUsableWidth(availableWidth, 1)} />}
-          {this.props.panelsToDisplay.includes("tree") ? <Tree width={big.width} height={big.height} key={keyName} /> : null}
-          {this.props.panelsToDisplay.includes("map") ? <Map width={big.width} height={big.height} key={keyName+"_map"} justGotNewDatasetRenderNewMap={false} legend={this.shouldShowMapLegend()} /> : null}
+          {this.props.panelsToDisplay.includes("tree") ?
+            <Tree
+              width={this.inGrid() ? grid.width : full.width}
+              height={this.inGrid() ? grid.height : full.height}
+              key={keyName}
+            /> :
+            null
+          }
+          {this.props.panelsToDisplay.includes("measurements") && this.props.measurementsLoaded ?
+            <Suspense fallback={null}>
+              <Measurements
+                width={this.inGrid() ? grid.width : full.width}
+                height={this.inGrid() ? grid.height : full.height}
+                key={keyName+"_measurements"}
+                showLegend={this.shouldShowMeasurementsLegend()}
+              />
+            </Suspense> :
```

Comment:
I noticed that the measurements panel takes a couple seconds to pop in, so I first see the tree adjacent to the map and then suddenly the map jumps and is replaced by the measurements panel. I think this is due to the condition on `this.props.measurementsLoaded` here and/or the `null` Suspense fallback? Are there alternatives that would be less disruptive during page load?

Reply:
Yeah, this behavior is not my favorite either. 

We can include the measurements panel on load but display the Nextstrain spinner while the sidecar data is loading to be less disruptive. Would it be disruptive to then remove the panel if there was an error loading the data? Or do we let the empty panel (maybe an error message) sit there and leave it to the user to toggle off the panel?

Reply:
I may not have full context here, but: if the measurements panel has to be toggled on (by the dataset's Auspice config? or by the user viewing the dataset?), then I'd say it's friendlier to display it with an error (or some indication of an error, at least) if the data can't be loaded rather than quietly not show it. Otherwise it may seem as if the config/toggle isn't working?

Reply:
![Screen Shot 2022-02-16 at 3 27 46 PM](https://user-images.githubusercontent.com/40774762/154375077-b7cb1187-ee81-4885-a4b2-08b3d9393f70.png)
Currently, when the sidecar JSON isn't able to load property, a warning notification pops up. There's no disruption in the rest of the page because the measurements panel itself was never rendered due to the condition on `this.props.measurementsLoaded`. 

However, given the expectation that the sidecar JSON should load properly more often than not, I see that we should reverse the disruptive behavior. The panel should be rendered on page load (with a fallback spinner waiting for data), and then removed if the data doesn't load properly. There would be a disruption but I think an expected one since there's a warning notification as well.